App.js

```diff
@@ -1,26 +1,114 @@
-import React from "react"
+import React, { useEffect, useState } from "react";
 import styled from "styled-components/native"
+import Header from "./components/Header"
+import {
+  Image,
+  View,
+  TouchableOpacity,
+  Text,
+  ScrollView,
+  Linking,
+  Share
+} from 'react-native';
+
+const apiKey = `563492ad6f917000010000016674d16c530e444482c459f1837b2a47`
+const url = "https://api.pexels.com/v1/search?query=example+wildlife&per_page=15&page=1"
 
 const App = () => {
+  const [photos, setPhotos] = useState([])
+
+  useEffect(() => {
+    fetch(url, { headers: { Authorization: apiKey } })
+      .then(res => res.json())
+      .then(json => {
+        setPhotos(json.photos)
+        console.log(json)
+      })
+  }, [])
+
+  const getImageToShare = photoId => {
+    const photo = photos.find(p => p.id === photoId)
+    return photo.src.original
+  }
+
+  const onShare = async photoId => {
+    const result = await Share.share({
+      message: 'Hi there, isnt this just awsome?',
+      url: getImageToShare(photoId),
+      title: 'Wildlife bliss'
+    })
+
+    if (result.action === Share.sharedAction) {
+      if (result.activityType) {
+        //shared with activity type of result.activityType
+        alert('some wildlife inspo shared!')
+      } else {
+        //shared
+      }
+    } else if (result.action === Share.dismissedAction) {
+      //dismissd
+      alert('dismissed')
+    }
+  }
+
+  const getSelectedImageUrl = (photoId) => {
+    const photo = photos.find((p) => p.id === photoId)
+    return Linking.openURL(photo.photographer_url)
+  }
+
   return (
     <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
-    </Container>
+      <Header title='Welcome to explore the wildlife!' />
+      <ScrollView>
+        {photos.map((photo) => (
+          <View key={photo.id}>
+
+            <TouchableOpacity onPress={() => getSelectedImageUrl(photo.id)}>
+              <Image
+                source={{ uri: photo.src.original }}
+                style={{ width: 400, height: 300, marginVertical: 10, }} />
```

Comment:
try to not use styling inline like this.
Rather go with StyleSheet or Styled components.